AppManager/app_manager_server.py

```diff
@@ -621,17 +623,14 @@ def remove_conflicting_jars(app_name):
     logging.warn("Lib directory not found in app code while updating.")
     return
   logging.info("Removing jars from {0}".format(lib_dir))
-  subprocess.call("rm -f " + lib_dir + \
-    "/appengine-api-1.0-sdk-*.jar", shell=True)
-  subprocess.call("rm -f " + lib_dir + \
-    "/appengine-api-stubs-*.jar", shell=True)
-  subprocess.call("rm -f " + lib_dir + \
-    "/appengine-api-labs-*.jar", shell=True)
-  subprocess.call("rm -f " + lib_dir + \
-    "/appengine-jsr107cache-*.jar", shell=True)
-  subprocess.call("rm -f " + lib_dir + \
-    "/jsr107cache-*.jar", shell=True)
-
+  file_pattern = ['appengine-api-1.0-sdk-*.jar', 'appengine-api-stubs-*.jar',
```

Comment:
`file_pattern` is a bit generic. Without looking at the logic, I can't tell at a glance that these are going to be removed. Maybe `conflicting_jars`?